packages/editor/src/lib/config/createTLStore.ts

```diff
@@ -55,12 +54,40 @@ export const defaultAssetStore: TLAssetStore = {
 	resolve: (asset) => asset.props.src,
 }
 
+/**
+ * A helper for creating a TLStore schema.
+ *
+ * @param opts - Options for creating the schema.
+ *
+ * @public
+ */
+export function createTLStoreSchema(
```

Comment:
I'm a bit worried about the naming similarity to `createTLSchema` here. Can you think of a name that more clearly distinguishes the behaviour of this? (and the hook I guess)?